src/components/forms/formr-part-a/ProfileToFormRPartAInitialValuesMapping.ts

```diff
@@ -1,15 +1,24 @@
 import { TraineeProfile } from "../../../models/TraineeProfile";
 import { FormRPartA } from "../../../models/FormRPartA";
 
-export function mapProfileToFormRPartAInitialValues(
+export function ProfileToFormRPartAInitialValuesMapping(
   traineeProfile: TraineeProfile | null
 ): FormRPartA | null {
   if (!traineeProfile) {
     return null;
   }
 
   const pd = traineeProfile.personalDetails;
-  const programme = traineeProfile.programmeMemberships[0];
+  const programme = traineeProfile.programmeMemberships
+    .filter(pm => new Date(pm.startDate) < new Date())
```

Comment:
Just remembered Ashley told us the programmeMemberships could be future ones.

Reply:
Ya, for my understanding, the programmeMemberships can be current or future. Just compare to find the latest startDate should be alright. This line may not be needed.

Reply:
If I did not misunderstand Ashley, I think the programmes can be 'not yet started'. That is Yafang's meaning of 'future programme'.

Reply:
We're basically look for programmes with an end date later than the current date, yes?

Reply:
@hihilary & @DorisWongNHS Ash confirmed that we have to consider future programmes as well(Applies the same as in curriculum as well). I will update the conditions accordingly.